CONTRIBUTING.md

```diff
@@ -98,6 +100,7 @@ Once you're ready to contribute:
 6. Use Jekyll to [build a version of the site locally](#build-and-test-the-docs-locally) so you can view your changes in a browser:
 
     ```
+    cd src/<docs version to build>
```

Comment:
Today, I think the only thing you can build is `src/current`, right?

Reply:
Today, you can build in `current`, `api`, and `interactive-tutorials`.